crates/metaslang/bindings/src/builder/loader/mod.rs

```diff
@@ -386,18 +386,16 @@ pub(crate) struct Loader<'a, KT: KindTypes + 'static> {
     graph: Graph<KT>,
     remapped_nodes: HashMap<usize, NodeID>,
     injected_node_count: usize,
-    cursors: HashMap<Handle<Node>, Cursor<KT>>,
     definitions_info: HashMap<Handle<Node>, DefinitionBindingInfo<KT>>,
-    references_info: HashMap<Handle<Node>, ReferenceBindingInfo>,
+    references_info: HashMap<Handle<Node>, ReferenceBindingInfo<KT>>,
```

Comment:
It will be interesting to see the effect of removing this hashmap (and the clones inside it) in memory usage..